src/test/java/uk/gov/pay/adminusers/service/ServiceOtpDispatcherTest.java

```diff
@@ -52,6 +64,59 @@ public void shouldSuccess_whenDispatchServiceOtp_ifInviteEntityExist() {
         assertThat(dispatched,is(true));
     }
 
+    @Test
+    public void shouldSuccess_whenDispatchServiceOtp_ifInviteEntityExist_butPhoneAndPasswordOnlyInRequest_andUpdateInviteEntityWithPhoneAndPassword() {
```

Comment:
```suggestion
    public void shouldSucceed_whenDispatchServiceOtp_ifInviteEntityExist_butPhoneAndPasswordOnlyInRequest_andUpdateInviteEntityWithPhoneAndPassword() {
```
Minorest of minors...

Reply:
This offends my grammatical sensibilities too but it’s consistent with the naming of the other tests.

Reply:
wouldn't be too much to change other tests too...